src/webthings-client.ts

```diff
@@ -5,10 +5,13 @@
  */
 
 import fetch, { RequestInit } from 'node-fetch';
+import { EventEmitter } from "events";
+import { client as WebSocketClient } from "websocket";
```

Comment:
Single quotes :grimacing: 

```suggestion
import { EventEmitter } from 'events';
import { client as WebSocketClient } from 'websocket';
```